### PR TITLE
Feature/identifier

### PR DESCRIPTION
**Pull request by scottnath**
Identifier for content types. Allows us to indicate which attribute should be used to _identify_ this piece of content.

---
Resolves https://github.com/punchcard-cms/punchcard/issues/143, #72

`DCO 1.1 Signed-off-by: Scott Nath <github@scottnath.com>`

### CODE REVIEW

**Comment by Snugug** [comment]
@scottnath Branch is out of date

**Comment by scottnath** [comment]
@Snugug updated.

File: lib/content-types/merge.js [hunk 1]
```diff
@@ -155,6 +169,27 @@ const squish = (types) => {
         return plugin;
       });
 
+      // find attribute which is the identifier
+      const contender = attrs.find(attr => {
```
Comment (Snugug) [inline lib/content-types/merge.js]: Excellent. Didn't know `find` exists, can probably update `singleItem` with this

**Comment by Snugug** [comment]
Thanks, perfect!

**Pull request merged**